### PR TITLE
Removed PreCheck Step

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Removed PreCheck Step from "Run Scenario" Playbook

### CODE REVIEW

File: playbooks/02 - Use Case - SOC Simulator/Run Scenario - Create Alerts.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Run Scenario - Create Alerts",
     "aliasName": null,
     "tag": null,
-    "description": null,
+    "description": "Creates records related to scenario",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - SOC Simulator/Run Scenario - Create Alerts.json]: Create*

File: playbooks/02 - Use Case - SOC Simulator/Run Selected Scenario.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Run Selected Scenario",
     "aliasName": null,
     "tag": null,
-    "description": null,
+    "description": "Execute a scenario and create its related records",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - SOC Simulator/Run Selected Scenario.json]: Execute a scenario and create its related records --> Execute scenario to create respective records? OR Execute a scenario and create related records

File: playbooks/02 - Use Case - SOC Simulator/Run Scenario - Create Alerts.json (hunk 1 above)
Comment (cs-rohit-khune) [reply playbooks/02 - Use Case - SOC Simulator/Run Scenario - Create Alerts.json]: @cs-suyog-jinturkar  Updated PB description to 'Create records related to a scenario'

File: playbooks/02 - Use Case - SOC Simulator/Run Selected Scenario.json (hunk 2 above)
Comment (cs-rohit-khune) [reply playbooks/02 - Use Case - SOC Simulator/Run Selected Scenario.json]: @cs-suyog-jinturkar Updated PB description to "Execute a scenario and create related records"

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**